### PR TITLE
Feature/vr 263

### PR DESCRIPTION
**Pull request by n3op2**
# Pull Request

## Checklist
- [x] Have you read Digital Catapult's [Code of Conduct](https://github.com/digicatapult/.github/blob/main/CODE_OF_CONDUCT.md)?
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.

## PR Type

Please delete options that are irrelevant.

- [x] Feature

## Linked tickets

https://digicatapult.atlassian.net/browse/VR-263

## High level description

New admin invite email template with the new copy

## Detailed description

Changes:
- unit test updates
- admin email template update
- screenshot comparison for homepage

## Operational impact

Low/None

### CODE REVIEW

File: src/models/emailServiceInt/templates/invitationAdmin.tsx [hunk 1]
```diff
@@ -1,23 +1,40 @@
...
+          <h1>{Html.escapeHtml(`Hi ${localCompany.company_name}`)}</h1>
+          <br />
+          <p>
+            {Html.escapeHtml(
+              `${params.invitator} has sent you a request to connect securely on Veritable. To complete the verification process, please enter the following 6-digit postal code within your Veritable instance.`
+            )}
+          </p>
+
+          <br />
+          <p>Verification Code:</p>
```
Comment (mattdean-digicatapult) [inline src/models/emailServiceInt/templates/invitationAdmin.tsx:37]: Needs updating. See latest copy on ticket:

```
Hi {Alice’s Company} Admin

Please post this verification code to {Bob’s Company}, to complete the verification process.

Address:
[Postal address of Bob’s Company]

Verification Code:
[123456]
```

File: test/snapshots/a-homepage.spec.ts/Homepage-and-homepage-related-tests-after-successful-login-renders-a-home-page-1.png
Comment (mattdean-digicatapult) [inline test/snapshots/a-homepage.spec.ts/Homepage-and-homepage-related-tests-after-successful-login-renders-a-home-page-1.png:1]: duplicate?

File: test/snapshots/a-homepage.spec.ts/Homepage-and-homepage-related-tests-redirects-to-keycloak-1.png
Comment (mattdean-digicatapult) [inline test/snapshots/a-homepage.spec.ts/Homepage-and-homepage-related-tests-redirects-to-keycloak-1.png:1]: duplicate?

File: test/snapshots/a-homepage.spec.ts/Homepage-and-homepage-related-tests-after-successful-login-renders-a-home-page-1.png
Comment (n3op2) [reply test/snapshots/a-homepage.spec.ts/Homepage-and-homepage-related-tests-after-successful-login-renders-a-home-page-1.png:1]: indeed, has been removed

File: test/snapshots/a-homepage.spec.ts/Homepage-and-homepage-related-tests-redirects-to-keycloak-1.png
Comment (n3op2) [reply test/snapshots/a-homepage.spec.ts/Homepage-and-homepage-related-tests-redirects-to-keycloak-1.png:1]: removed as well

File: src/models/emailServiceInt/templates/invitationAdmin.tsx [hunk 2]
```diff
@@ -1,23 +1,40 @@
...
+          <br />
+          <p>
+            {Html.escapeHtml(
+              `${params.invitator} as sent you a request to connect securely on Veritable. To complete the verification process, please enter the following 6-digit postal code within your Veritable instance.`
+            )}
+          </p>
+
+          <br />
+          <p>Verification Code:</p>
           <p>{Html.escapeHtml(params.pin)}</p>
```
Comment (mattdean-digicatapult) [inline src/models/emailServiceInt/templates/invitationAdmin.tsx:38]: This still doesn't match the latest copy in the ticket. Please look again

Comment (n3op2) [reply src/models/emailServiceInt/templates/invitationAdmin.tsx:38]: there is a comment that I've used as a refecrence -> https://digicatapult.atlassian.net/browse/VR-263?focusedCommentId=107293

Comment (n3op2) [reply src/models/emailServiceInt/templates/invitationAdmin.tsx]: It looks like I was using the wrong comment. Confirmed and updated with the correct one.

File: src/models/emailServiceInt/templates/invitationAdmin.tsx [hunk 3]
```diff
@@ -1,23 +1,41 @@
...
+          <h1>{Html.escapeHtml(`Hi ${localCompany.company_name}`)}</h1>
+          <br />
+          <p>
+            {Html.escapeHtml(
+              `Please post this verification code to ${params.receiver}, to complete the verification process.`
+            )}
+          </p>
+
+          <br />
+          <p>Verification Code:</p>
```
Comment (mattdean-digicatapult) [inline src/models/emailServiceInt/templates/invitationAdmin.tsx:38]: The copy between the subject and body is logically mixed up. When I do an invitation from alice to bob I get

```
Subject: Postal Code for Verification: Invitation from OFFSHORE RENEWABLE ENERGY CATAPULT on Veritable

Hi DIGITAL CATAPULT Admin,
Please post this verification code to OFFSHORE RENEWABLE ENERGY CATAPULT, to complete the verification process.

Address: OFFSHORE RENEWABLE ENERGY CATAPULT, Offshore House, Albert Street, Blyth, NE24 1LZ, Northumberland

Verification Code: 856478
```

In the `Subject` this should either be:

1. `Postal Code for Verification: Invitation from DIGITAL CATAPULT on Veritable`
2. `Postal Code for Verification: Invitation to OFFSHORE RENEWABLE ENERGY CATAPULT on Veritable`

in order to logically make sense. I think you need to ask Esther which is preferable.

Comment (n3op2) [reply src/models/emailServiceInt/templates/invitationAdmin.tsx:38]: oops, yea, currently used receiver rather than local, have been updated.

**Review by mattdean-digicatapult — APPROVED** [review]

**Pull request merged**